apps/dashboard/app/controllers/transfers_controller.rb

```diff
@@ -17,7 +17,18 @@ def show
 
   def create
     body_params = JSON.parse(request.body.read).symbolize_keys
-    @transfer = Transfer.build(action: body_params[:command], files: body_params[:files])
+
+    from_fs = body_params.fetch(:from_fs, "fs")
+    to_fs = body_params.fetch(:to_fs, "fs")
+
+    if from_fs == "fs" && to_fs == "fs"
```

Comment:
We can defer this - but the string `"fs"` is around enough that we likely need to make it a constant and refer to it instead of hoping we type `fs` correctly all over the place.

Reply:
That would be a good idea. `"fs"` definitely is used in quite a few different places.
What would be the correct place for that constant? RcloneUtil?

Reply:
That seems as good as any.

Reply:
Added the constant `RcloneUtil::LOCAL_FS_NAME`. Only changed the places where this PR adds `"fs"`.

Reply:
Strange, tests ran perfectly fine locally and worked fine in browser too, but somehow it's broken now. Will fix.

Reply:
Should be good now, tests passed locally since the tests using `RemoteFile` ran first when I ran them locally, and same situation in browser.
https://github.com/OSC/ondemand/compare/bc733533fa94f53440ca97a6d74d8f51b6508e28..7f8652e21a65b205835b1af107fe0acd213574e5